components/Faq.vue

```diff
@@ -141,7 +141,7 @@ export default {
   margin: 10px auto;
   text-align: left;
   outline: none;
-  transition: 0.4s;
+  transition: 0.2s;
```

Comment:
I don't really like the transition. Looks buggy to me. I suggest either fixing it so it's not buggy or removing it altogether

The box when it appears is cut off from the original button, which is why it looks buggy. Chrome desktop

Reply:
Design asked us to change a lot about the FAQ, which hasn't been implemented yet. This should look better once the changes are made.

Reply:
if you run this branch it has the new changes im pretty sure

Reply:
![image](https://user-images.githubusercontent.com/11839441/65377437-a681b400-dc60-11e9-9563-a0cbff000ca3.png)

Reply:
oop you right, my bad